base/strings/basic.jl

```diff
@@ -100,7 +100,40 @@ function length(s::AbstractString)
 end
 
 ## string comparison functions ##
+"""
+    cmp(a::AbstractString, b::AbstractString)
+
+Compares two strings for equality. Returns `0` if both strings have the same
```

Comment:
Just a general comment: docstrings should generally use imperative form, see point 2 here: https://docs.julialang.org/en/latest/manual/documentation/#Documentation-1 so `Compares` -> `Compare` etc